renovate.json

```diff
@@ -1,12 +1,18 @@
 {
   "rangeStrategy": "replace",
   "extends": ["config:base"],
-  "includesPeerDependencies": true,
   "packageRules": [
     {
+      "matchDepTypes": ["dependencies", "devDependencies"],
```

Comment:
Should we omit this line so it goes back to the defaults for this rule?

Reply:
Yes!